packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.2.11
+
+* Fixes swift compressed list data types.
```

Comment:
Nit: let's keep the item tagging that the channel for Pigeon has mostly used. So: `[swift] Fixes ...`